### PR TITLE
add comments

### PR DESCRIPTION
**Pull request by ZhouHansen**
Signed-off-by: ZhouHansen <z308114274@gmail.com>

This is a 🔦 documentation change

Add an example for `pub fn blocks()`. 

## Checklist

- [x] tests pass
- [ ] tests and/or benchmarks are included
- [x] documentation is changed or added

## Context
None

## Semver Changes
None

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -199,6 +199,27 @@ impl TreeIndex {
...
   /// NOTE: should we rename this to `.len()` ?
+  /// ## Examples
+  /// ```
+  ///        3
+  ///    1       5
+  ///  0   2   4   6  
+  /// ```
+  ///
+  /// ```rust
+  /// let mut tree = TreeIndex::default();
```
Comment (yoshuawuyts) [inline src/lib.rs]: Tests fail here with `Use of undeclared module TreeIndex`.

2018-09-03-184015_3840x2160

Comment (ZhouHansen) [reply src/lib.rs]: ah! fix it.

File: src/lib.rs [hunk 2]
```diff
@@ -200,13 +200,16 @@ impl TreeIndex {
   ///
   /// NOTE: should we rename this to `.len()` ?
   /// ## Examples
-  /// ```
+  /// ```readme
```
Comment (yoshuawuyts) [inline src/lib.rs]: `txt` might be more accurate, but don't mind this either

**Comment by ZhouHansen** [comment]
Thanks a lot.

**Pull request closed without merging**